packages/nextjs/test/integration/test/server/tracingHttp.js

```diff
@@ -9,9 +9,7 @@ module.exports = async ({ url: urlBase, argv }) => {
   const url = `${urlBase}/api/http`;
 
   // this intercepts the outgoing request made by the route handler (which it makes in order to test span creation)
-  nock('http://example.com')
-    .get('/')
-    .reply(200, 'ok');
+  nock('http://example.com').get('/').reply(200, 'ok');
```

Comment:
This is interesting to me. Are we fine with that change? I might be wrong but to me it seems like this chain was spread over multiple lines on purpose. There are some similar reformats with arguments, array contents, etc. which were also reformatted to a single line.

I'm not criticising the change itself and I know that argument/parameter wrapping and alignment is hard to define properly for every situation but I just wanted to highlight that this _might_ be an unintended format change. Nevertheless, I think consistency is important so I'd be totally on board with the change.

Reply:
I actually happen to agree with you that it's easier to read on multiple lines. That said, it's not up to us, or at least, as long as we're using Prettier it's not - it will make the change whether we want it to or not (and that's not configurable - see below). And since we format on save, trying to keep it in a different state than Prettier wants it is ultimately a losing game. (I know, I've tried!) This PR just keeps those inevitable changes from ending up in unrelated PRs because someone happens to have touched one of these files and hit 'save.'

Here's Prettier's take on why it should get to choose:

https://prettier.io/docs/en/why-prettier.html
https://prettier.io/docs/en/option-philosophy.html

Reply:
Okay, thanks for the links. They're an interesting read. So as I've said, the PR is looking good to me.